packages/gatsby-plugin-page-creator/README.md

```diff
@@ -46,3 +50,57 @@ module.exports = {
   ],
 }
 ```
+
+### Ignoring Specific Files
+
+#### Shorthand
+
+```javascript
+// The following example will disable the `/blog` index page
+// Note: This will only stop the creation of the `/blog` page!
+// To disable both the `/blog` and `/blog/post-slug` pages,
+// make a config.js which defines an `ignorePages` array
```

Comment:
This is very valid note, but I think it comes too early (before showing basic example of `ignore` options in the plugin.

I think we need this note, but it should probably be placed after `gatsby-config` code snippet (?)

Reply:
Sure. Looking at this again, I wonder if the entire note is necessary, given that it details a specific implementation. Maybe the following would be more appropriate?

```md
**NOTE**: The above code snippet will only stop the creation of the `/blog` page. 
This plugin is only responsible for creating pages from React components. 
It does not affect the generation of pages from content/data files 
(see [gatsby-source-filesystem](https://www.gatsbyjs.org/packages/gatsby-source-filesystem/)).
```

Reply:
@pieh do you prefer the original note, or the new one regarding the scope of `ignore` for `gatsby-plugin-page-creator`?

Reply:
I think new one is better - we don't need to show opinionated way of doing global ignore that is used both by this plugin and in gatsby-node.

What about:
```
This plugin does not affect the programmatically generated pages with [`createPages` API](https://www.gatsbyjs.org/docs/node-apis/#createPages)
```

Reply:
Yeah, that's much clearer.